tests/api-watch-diff/test.ml

```diff
@@ -1,4 +1,400 @@
+open Types
+open Predef
```

Comment:
Let's use qualified access to `Predef` rather than opening it, so we can see exactly where we are pulling these values from. So that means removing this and anytime we need a value from it we can do, e.g., `Predef.type_int`.

Reply:
Done